assets/src/setup/pages/summary/reader.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { ToggleControl } from '@wordpress/components';
+import { useContext } from '@wordpress/element';
+/**
+ * Internal dependencies
+ */
+import { Options } from '../../components/options-context-provider';
+import { Desktop } from '../../components/desktop';
+import { Phone } from '../../components/phone';
+import { ReaderThemes } from '../../components/reader-themes-context-provider';
+
+/**
+ * Summary screen when reader mode was selected.
+ *
+ * @param {Object} props
+ * @param {Object} props.currentTheme Data for the theme currently active on the site.
+ */
+export function Reader( { currentTheme } ) {
+	const { options, updateOptions } = useContext( Options );
+	const { themes } = useContext( ReaderThemes );
+
+	const { reader_theme: readerTheme, mobile_redirect: mobileRedirect } = options;
+
+	const readerThemeData = themes ? themes.find( ( theme ) => theme.slug === readerTheme ) : {};
+
+	return (
+		<div className="reader-summary">
+			<p className="reader-summary__description">
+				{ __( 'In Reader mode your site will have a non-AMP and an AMP version, and each version will use its own theme. If automatic mobile redirection is enabled, the AMP version of the content will be served on mobile devices. If AMP-to-AMP linking is enabled, once users are on an AMP page, they will continue navigating your AMP content.', 'amp' ) }
+			</p>
+			<ToggleControl
+				label={ __( 'Redirect mobile visitors to AMP', 'amp' ) }
+				help={ __( 'AMP is not only for mobile.', 'amp' ) }
+				checked={ true === mobileRedirect }
+				onChange={ () => {
+					updateOptions( { mobile_redirect: ! mobileRedirect } );
+				} }
+			/>
+
+			<div className="reader-summary__screens">
+				<div>
+					<div className="selectable selectable--bottom">
+						<Desktop>
+							<img src={ currentTheme.screenshot } alt={ currentTheme.name } />
```

Comment:
Please add dimensions for the image to avoid content shifting. Also add `loading=lazy` and `decoding=async`.